test/active_record_test.rb

```diff
@@ -364,6 +364,15 @@ class Measured::Rails::ActiveRecordTest < ActiveSupport::TestCase
     assert_equal custom_unit_thing.extra_weight, Measured::Weight.new(12, :kg)
   end
 
+  test "using custom value fields works correctly" do
```

Comment:
This test case has no asserts. It implicitly checks that there are no raises. The test above this line `test "changing unit value when shared affects all fields" do` does some `assert_equal` to make sure the values we get back are expected. Can you update this test to be more like that?